src/app/services/property-detail/property-detail.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@angular/core";
+import { HttpHeaders } from "@angular/common/http";
+import { Observable } from "rxjs";
+import { HttpService } from "../http.service";
+import { LocalStorageService } from '../local-storage.service'
+
+@Injectable({
+    providedIn: "root"
+})
+export class PropertyDetailService {
+    userToken = this.localStorageService.get('token', '');
+
+    constructor(
+        private http: HttpService,
+        private localStorageService: LocalStorageService) { 
+        
+        };
+
+    getProperty(slug): Observable<any> {
+
+        const httpOptions = {
+            headers: new HttpHeaders({
+            Authorization: `Bearer ${this.userToken}`
+
```

Comment:
Thanks for the great work @dorothyas , kI can see you implement httpHeaders with user token but I think Alpha already created interceptors that do this.

Reply:
@Genza999 I have worked on this

Reply:
Cool!